rest-api/tools/setup_database.sh

```diff
@@ -100,36 +91,50 @@ function finish {
 }
 trap finish EXIT
 
-echo '{"db_connection_string": "'$CONNECTION_STRING'", ' \
-     ' "backup_db_connection_string": "'$BACKUP_CONNECTION_STRING'", '\
-     ' "rdr_db_password": "'$RDR_PASSWORD'", ' \
-     ' "root_db_password": "'$ROOT_PASSWORD'", ' \
-     ' "read_only_db_password": "'$READONLY_PASSWORD'", ' \
-     ' "db_connection_name": "'$INSTANCE_CONNECTION_NAME'", '\
-     ' "backup_db_connection_name": "'$BACKUP_INSTANCE_NAME'", '\
-     ' "db_user": "'$RDR_DB_USER'", '\
-     ' "db_name": "'$DB_NAME'" }' > $TMP_DB_INFO_FILE
+function get_existing_db_config {
+  "Getting database info..."
+  tools/install_config.sh --key db_config --instance $INSTANCE \
+   --creds_file ${CREDS_FILE} --config_output "$TMP_DB_INFO_FILE"
+}
 
 for db_name in "rdr" "metrics"; do
```

Comment:
So I realized the logic before had a bug here, which we should fix while we're at this... the only time we need to loop through these DBs is specifically when running create_db.sql. For updating passwords there's no point in running things twice. So you can put this loop inside the else block.

Reply:
done